README.md

```diff
@@ -134,10 +134,49 @@ services:
     - ...
 
 ```
+### Start with a distro backup
+
+To restore the backup unzip the the content and proceed with the following steps:
```

Comment:
Could you describe a bit more what backup.zip we are talking about, and what it mostly contains?